stanford_profile.post_update.php

```diff
@@ -111,11 +111,13 @@ function stanford_profile_post_update_8015() {
  */
 function _stanford_profile_react_paragraph_fix() {
   $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+  $time = strtotime('September 14 2020, 11:59 PM');
```

Comment:
UTC?

Reply:
I figured its a rough value. close enough kinda thing

Reply:
That is 8 hours off... so that could have gotten into when people are up and editing. Let me check our known sites for their times.

Reply:
Roughly 3:00pm on the known sites.  Think we are ok.

Reply:
yeah and it might use UTC. Acquia does it a little different sometimes

Reply:
```suggestion
  $time = strtotime('September 14 2020, 11:59 PM PDT');
```

Reply:
that'll work

Reply:
FWIW `drush @pookmish20191003.01live eval 'echo strtotime("September 14 2020, 11:59 PM");'` returned a timestamp for 5PM PDT. so it would've been about the same either way